### PR TITLE
Remove winzou state machine dependency

### PR DESCRIPTION
**Pull request by loic425**
| Q               | A
| --------------- | -----
| Bug fix?        | no
| New feature?    |yes
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

### CODE REVIEW

**Comment by loic425** [comment]
@pamil This one is ready, I will make the second part (upgrading the test app) on another PR.

File: src/Bundle/test/app/AppKernel.php [hunk 1]
```diff
@@ -16,33 +16,26 @@
...
-        return $bundles;
     }
 
     public function registerContainerConfiguration(LoaderInterface $loader): void
     {
         $loader->load(__DIR__ . '/config/{config}.{php,xml,yaml,yml}', 'glob');
         $loader->load(__DIR__ . '/config/{config}_' . $this->getEnvironment() . '.{php,xml,yaml,yml}', 'glob');
     }
+
+    private function addBundleOnlyIfPresent(string $bundleClassName): void
```
Comment (loic425) [inline src/Bundle/test/app/AppKernel.php:35]: oops, I have to remove this.

File: composer.json [hunk 2]
```diff
@@ -40,16 +40,16 @@
...
     },
     "replace": {
         "sylius/resource": "self.version"
     },
     "conflict": {
         "friendsofsymfony/rest-bundle": "<3.0 >=4.0",
         "jms/serializer-bundle": "<3.5 >=4.0",
-        "willdurand/hateoas-bundle": "<2.0 >=3.0"
+        "willdurand/hateoas-bundle": "<2.0 >=3.0",
+        "winzou/state-machine-bundle": "<0.3.2 || >=0.4.0 <0.4.3"
```
Comment (loic425) [inline composer.json:52]: @pamil I'm not sure at all for "<0.3.2 || >=0.4.0 <0.4.3", Can you confirm it?

File: src/Bundle/test/app/AppKernel.php [hunk 3]
```diff
@@ -16,28 +16,14 @@
 
 class AppKernel extends Kernel
 {
-    public function registerBundles()
+    public function registerBundles(): iterable
```
Comment (loic425) [inline src/Bundle/test/app/AppKernel.php:19]: @pamil I just use that part of new Symfony architecture to have a something easier with all environments on travis.

File: composer.json (hunk 2 above)
Comment (pamil) [reply composer.json:52]: Should be fine I guess 👍

**Review by pamil — APPROVED** [review]

**Review by pamil — CHANGES_REQUESTED** [review]
`src/Bundle/test/config/db.sql` should not be added

**Comment by pamil** [comment]
Rebased with master and ported Travis CI changes to GitHub Actions 🎉

**Comment by pamil** [comment]
Thank you, Loïc! :1st_place_medal:

**Pull request merged**